cmake/SSGCommon.cmake

```diff
@@ -1242,19 +1242,19 @@ macro(ssg_define_guide_and_table_tests)
                 COMMAND "${GREP_EXECUTABLE}" "-zv" "</thead>[[:space:]]*</table>" "${TABLE_FILE}"
             )
         endforeach()
-        add_test(
-            NAME "unique-cces"
-            COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tests/assert_reference_unique.sh" "cce"
-        )
-        # Disable unique-stigids temporarily since it breaks many jobs. There are multiple duplicates
-        # of STIG IDs and they should be fixed by DISA first.
-        # add_test(
-        #     NAME "unique-stigids"
-        #     COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tests/assert_reference_unique.sh" "stigid"
-        # )
-        set_tests_properties("unique-cces" PROPERTIES LABELS quick)
-        # set_tests_properties("unique-stigids" PROPERTIES LABELS quick)
     endif()
+    add_test(
+        NAME "unique-cces"
+        COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tests/assert_reference_unique.py" "${CMAKE_SOURCE_DIR}" "cce"
+    )
```

Comment:
I seems that the test now doesn't work. I have tried to make some duplicate CCEs and run the test. It prints the results but the result is pass even if they should fail.

```
[jcerny@thinkpad build{pr/10348}]$ ctest --verbose -R unique-cces
UpdateCTestConfiguration  from :/home/jcerny/work/git/scap-security-guide/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/jcerny/work/git/scap-security-guide/build/DartConfiguration.tcl
Test project /home/jcerny/work/git/scap-security-guide/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 4
    Start 4: unique-cces

4: Test command: /home/jcerny/work/git/scap-security-guide/tests/assert_reference_unique.py "/home/jcerny/work/git/scap-security-guide" "cce"
4: Working Directory: /home/jcerny/work/git/scap-security-guide/build
4: Test timeout computed to be: 10000000
4: 
4: cce CCE-84079-3 is included in files: 
4:  - linux_os/guide/system/selinux/selinux_policytype/rule.yml
4:  - linux_os/guide/system/selinux/selinux_state/rule.yml
1/1 Test #4: unique-cces ......................   Passed    0.24 sec

The following tests passed:
	unique-cces

100% tests passed, 0 tests failed out of 1

Label Time Summary:
quick    =   0.24 sec*proc (1 test)

Total Test time (real) =   0.25 sec
```

Reply:
Result code was not implemented. I'll add it and also split main into functions so codeclimate is happier.

Reply:
Okay, now it passes this edit,
```
1/6 Test  #77: unique-cces.......................***Failed    1.05 sec

cce CCE-84020-7 is included in files: 
 - applications/openshift/api-server/api_server_audit_log_path/rule.yml
 - applications/openshift/api-server/api_server_auth_mode_no_aa/rule.yml
 - applications/openshift/api-server/api_server_auth_mode_node/rule.yml

6/6 Test #283: missing-cces-rhel8
83% tests passed, 1 tests failed out of 6

Label Time Summary:
quick    =  11.08 sec*proc (6 tests)

Total Test time (real) =   2.36 sec

The following tests FAILED:
	 77 - unique-cces (Failed)
Errors while running CTest
```